RELEASE-NOTES.txt

```diff
@@ -1,5 +1,7 @@
 *** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
 
+* [*] On self-hosted sites, allow previewng unsaved changes to draft posts [https://github.com/wordpress-mobile/WordPress-Android/pull/16296]
```

Comment:
np: small typo previewng vs previewing

Reply:
Thanks for catching that! 🙇  Addressed in [df4d1fa](https://github.com/wordpress-mobile/WordPress-Android/pull/16296/commits/df4d1fa0709b4f16d3dd1b3a27fa5e884f59f1f0)